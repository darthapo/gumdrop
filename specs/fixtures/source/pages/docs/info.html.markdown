# I got stuff to say.

Are you going to listen?

Visit <http://zoodotcom.com> for fun and profit!