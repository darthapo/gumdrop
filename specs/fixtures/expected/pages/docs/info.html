<body class="site"><h1>I got stuff to say.</h1>

<p>Are you going to listen?</p>

<p>Visit <a href="http://zoodotcom.com">http://zoodotcom.com</a> for fun and profit!</p>
</body>